README.md

```diff
@@ -244,6 +244,10 @@ To speed-up a build you can reuse the binaries already downloaded from a previou
 snap install microk8s_v1.10.3_amd64.snap --classic --dangerous
 ```
 
+## How to Use MicroK8s
```

Comment:
I'd change this to "Who's Using MicroK8s"

Reply:
Done